image-sync/oc-mirror/mirror.sh

```diff
@@ -1,5 +1,15 @@
 #!/bin/sh
 az login --identity -u ${AZURE_CLIENT_ID}
-echo ${IMAGE_SET_CONFIG} | base64 -d | yq eval -P > /config/imageset-config.yaml
 DOCKER_COMMAND=/usr/local/bin/docker-login.sh az acr login -n ${REGISTRY}
-/usr/local/bin/oc-mirror --continue-on-error --config /config/imageset-config.yaml docker://${REGISTRY_URL} @$
+
+IMAGE_SET_CONFIG_FILE="/config/imageset-config.yaml"
+echo ${IMAGE_SET_CONFIG} | base64 -d | yq eval -P > ${IMAGE_SET_CONFIG_FILE}
+API_VERSION=$(yq eval '.apiVersion' ${IMAGE_SET_CONFIG_FILE})
+
+if echo "$API_VERSION" | grep -q "^mirror.openshift.io/v2"; then
+    ADDITIONAL_FLAGS="--workspace file:///oc-mirror-workspace --v2"
+else
+    ADDITIONAL_FLAGS="--continue-on-error"
```

Comment:
How is `--continue-on-error` conditional to the API version? Shouldn't it be always set?